api/python/provisioner/config.py

```diff
@@ -229,7 +229,7 @@ def profile_base_dir(
     setup_name: Optional[str] = 'default'
 ):
     if location is None:
-        location = Path.cwd() / PROFILE_DIR_NAME
+        location = Path('/root') / PROFILE_DIR_NAME
```

Comment:
@SomwanshiAnjali it will block all deployments for non root users. Please use home directory instead: it will respect both cases.

Reply:
User environment variable HOME.

Reply:
I beleive  pathlib can provide a better alternative: https://docs.python.org/3.6/library/pathlib.html#pathlib.Path.home

Reply:
Addressed